__tests__/Unit/hooks/tasksApi.test.tsx

```diff
@@ -30,11 +30,15 @@ function Wrapper({
 }: PropsWithChildren<Record<string, never>>): JSX.Element {
     return <Provider store={store()}>{children}</Provider>;
 }
-
+const dev = true;
 describe('useGetAllTasksQuery()', () => {
     test('returns all tasks', async () => {
         const { result, waitForNextUpdate } = renderHook(
-            () => useGetAllTasksQuery(),
+            () =>
+                useGetAllTasksQuery({
+                    dev: dev as unknown as boolean,
```

Comment:
Why are we doing `dev as unknown as boolean` ?

Reply:
done